llvm/lib/Bitcode/Reader/MetadataLoader.cpp

```diff
@@ -2323,7 +2323,7 @@ Error MetadataLoader::MetadataLoaderImpl::parseOneMetadata(
         GET_OR_DISTINCT(DIObjCProperty,
                         (Context, getMDString(Record[1]),
                          getMDOrNull(Record[2]), Record[3],
-                         getMDString(Record[4]), getMDString(Record[5]),
+                         getMDString(Record[5]), getMDString(Record[4]),
```

Comment:
Possibly worth adding a comment, or using variable names or something here? On the other hand maybe that doesn't really add much value - I'll leave it up to you.